examples/client-logger/app-insights-logger/README.md

```diff
@@ -0,0 +1,70 @@
+# @fluid-example/app-insights-logger
+
+## Overview
+
+This package provides a simple Fluid application complete with a UI view in [React](https://react.dev/) to test the Fluid App Insights telemetry logger that will route typical Fluid telemetry to configured Azure App Insights.
+
+<!-- AUTO-GENERATED-CONTENT:START (README_INSTALLATION_SECTION:includeHeading=TRUE&devDependency=TRUE) -->
```

Comment:
Since this is an example app project, we don't presumably don't expect external consumers to install this. We should probably remove this section from this README.

Reply:
Some of the other sample apps use one of our pre-canned pragmas for instructions on how to run the app, though (with an optional argument for whether or not running requires tinylicious). I would probably use that instead.